concepts/eventing/performance evaluation/nats-jetstream/different-cloud-providers/results.md

```diff
@@ -0,0 +1,113 @@
+# Performance Evalation of Eventing on different Cloud Providers
+
+## Table of Contents
+- Test Setup
+- Failure Types
+- Test Results
+- Screenshots of Crashes
+
+## Test Setup
+* Kyma CLI version: `2.6.2`
+* Kyma: 
+  * Production Profile
+  * JetStream with File Storage
+* K8s cluster:
+  * Kubernetes v1.24.3, v1.24.6
+  * Gardener cluster [Nodes: 2(min) to 6(max)]
+
+* Kyma deploy command:
+    ```
+    kyma deploy --source=main --profile production ...
+    ```
+* Testing tool: [K6](https://k6.io/) (deployed in the same Kyma cluster)
+
+## Failure Types:
+Here are the common crash failures occured during the testing:
+- F1  - Only one of the NATS Jetstream nodes storage is full
+- F2  - All JetStream node storage is full
+- F3  - NATS JetStream just doesn’t accept and dispatch events despite storage is not full. This is solely observed on Azure
```

Comment:
Do we know how to reproduce these failures? If yes, please let's document them as well.